<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/pill.png</normaloff>images/pill.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nombre_edit">
     <property name="placeholderText">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Direccion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="direccion_edit">
     <property name="placeholderText">
      <string>Direccion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Numero de identidad</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="id_edit">
     <property name="placeholderText">
      <string>Numero de identidad</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="edad_spinner">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="minimum">
      <number>18</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Contrase√±a</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_edit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Sexo</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rb_masculino">
       <property name="text">
        <string>Masculino</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_femenino">
       <property name="text">
        <string>Femenino</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_otro">
       <property name="text">
        <string>Otro</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bt_registro">
       <property name="text">
        <string>Registrarte</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
